### PR TITLE
Run simulated hosts with unique `UUID`

### PR DESCRIPTION
**Pull request by lucasmrod**
This is a workaround for [#3201](https://github.com/fleetdm/fleet/issues/3201).

Add uniquely generated UUID to the simulated hosts docker containers.

### CODE REVIEW

**Comment by lucasmrod** [comment]
Tested running:
`./build/fleetctl preview --tag main --preview-config containers-with-unique-uuid`

Screen Shot 2021-12-10 at 15 40 41

**Comment by chiiph** [comment]
This PR should target `master`, so that when we merge, we can QA `master` and then when we are happy we can "release" with a PR towards `production`.

**Comment by lucasmrod** [comment]
Somebody needs to merge `production` into `master` (seems it's behind a few commits).

**Comment by lucasmrod** [comment]
Let me know if this is the way: https://github.com/fleetdm/osquery-in-a-box/pull/13

**Comment by lucasmrod** [comment]
@chiiph: Also, on the `README.md` it says:
```
The master branch is used by fleetctl before version 4.5.0 and should not change anymore except for critical fixes
```

**Comment by chiiph** [comment]
Hm... well, maybe I'm not running on the latest knowledge, but the point was to have an intermediate point where we can merge things without potentially breaking preview for all.

**Comment by lucasmrod** [comment]
+1 to using `master`, let's wait on @zwass to confirm and I can update the docs.

**Comment by zwass** [comment]
If we merge to `master`, we may break `fleetctl preview` for users on `fleetctl` versions prior to 4.5.0. As 4.5.0 is only just over a month old, it would be good to hold off on making any changes to `master` for another month or so. Can we merge to a different branch (say, `develop`) until then?

**Comment by mna** [comment]
+1 for what Zach mentioned for the `master` branch - I think we could even stop using it forever (unless we find a critical fix that needs to be applied to < 4.5.0 too) and test on a feature branch, then merge into production once it looks good.

**Comment by lucasmrod** [comment]
+1, added `develop` branch and updated instructions on the main `README.md`.

File: README.md
Comment (mna) [inline README.md:12]: :+1:

**Review by mna — APPROVED** [review]

File: README.md
Comment (chiiph) [inline README.md:10]: We can probably limit merges to this branch to zach or something like that, so nobody makes the mistake (except zach, I guess)

**Review by chiiph — APPROVED** [review]

**Comment by zwass** [comment]
Seemed like the best thing I could do was require 6 approving reviews (!) for `master` before merging to it. By the time we even have 6 people to do that, no one will be using fleetctl <4.5.0 😆

**Pull request merged**
